### ISSUE TITLE
Only seeing 1 eye rendered in Vive Pro

### ISSUE THREAD

**Issue by NITechLabs** [RenderHeads/UnityPlugin-AVProVideo#761]
**Describe the bug**
Installed the new AVPro Video player for Windows. Dit default setup. ie: Created a empty object and added the VideoPlayer to it. Added my video file to the source. Rest defaults.
Then created a new Canvas and added AVPro Video as a child UI object to it. All defaults.
Then set my canvas to World space and placed it relative to my camera. I disabled the tracking of the Vive headset for this.
Start the scene.
Now I see the video playing but only on my left eye. I cannot find what to do to enable both eyes.

**Your Setup (please complete the following information):**
Unity 2020.3.3
AVPro Video version:  2.1.0
Operating system version: Win 10 (up to date)
Device model:  HTC Vive Pro Eye
Video specs (resolution, frame-rate, codec, file size): Vive native running on a RTX2080Ti

**Comment by kahnivore** [comment]
Hi @NITechLabs 

You could try forcing stereo eye mode to StereoEye.Both as detailed [here](https://www.renderheads.com/content/docs/AVProVideo/articles/feature-arvrxr.html#forcing-stereo-eye-mode).

The following pages on the AVPro [shaders](https://www.renderheads.com/content/docs/AVProVideo/articles/usage-shaders.html) and [updating multi-pass stereo](https://www.renderheads.com/content/docs/AVProVideo/articles/component-update-multipass-stereo.html#update-multi-pass-stereo) might also be of use to you. 

Please let us know if this solves your issue.

Thanks!

**Comment by AndrewRH** [comment]
@NITechLabs  

As @kahnivore  mentioned above if you're using multi-pass rendering for XR then you will need to add the component UpdateMultiPassStereo to your scene, you can read about it in the docs [here](https://www.renderheads.com/content/docs/AVProVideo/articles/component-update-multipass-stereo.html).  Otherwise try changing your XR rendering method to single-pass.

Does that help?

Thanks,

**Comment by NITechLabs** [comment]
Hi!
Thanks for the help!
I'm using SinglePass (only running on Windows and Vive).
I will try the StereoEye.Both, but for the moment I switched approach an use the displaying on a mesh (quad) and that works fine.
What I did see in my particular scene (HDRP) was that my environment fog blocked part of my quad with the video material. Everything below my horizontal mid line in my view was blocked. I disabled the fog (don't need it when showing the video).
